### PR TITLE
Tweet organizing pipeline

### PR DESCRIPTION
**Pull request by stephen-lt-sl**

### CODE REVIEW

File: client/angular/services/columnAssignmentService.js [hunk 1]
```diff
@@ -0,0 +1,49 @@
+(function() {
+    angular
+        .module("TwitterWallApp")
+        .factory("columnAssignmentService", columnAssignmentService);
+
+    columnAssignmentService.$inject = [];
+
+    function columnAssignmentService($http) {
```
Comment (susanthomson) [inline client/angular/services/columnAssignmentService.js]: does this need http?

**Review by susanthomson — APPROVED** [review]

File: spec/client/services/columnAssignmentService.spec.js [hunk 2]
```diff
@@ -0,0 +1,80 @@
+require("phantomjs-polyfill");
```
Comment (susanthomson) [inline spec/client/services/columnAssignmentService.spec.js]: > > ReferenceError: Can't find variable: require at
> > ....\C:\dev\Bristol-Grads-2016\spec\client\services\columnAssignmentService.spec.js:1 global code

**Review by susanthomson — COMMENTED** [review]
:sweden:

**Review by susanthomson — CHANGES_REQUESTED** [review]
:sweden:

File: spec/client/services/columnAssignmentService.spec.js [hunk 3]
```diff
@@ -0,0 +1,80 @@
...
+        var x = new(columnAssignmentService.ColumnData)(2, function(tweet) {
+            return tweet.pinned === true;
+        }, function(tweetA, tweetB) {
+            return tweetA.pinTime - tweetB.pinTime;
+        });
+        testColumnDataList = [
+            new columnAssignmentService.ColumnData(2, function(tweet) {
+                return tweet.pinned === true;
+            }, function(tweetA, tweetB) {
+                return tweetA.pinTime - tweetB.pinTime;
```
Comment (BRHoward) [inline spec/client/services/columnAssignmentService.spec.js:29]: these sort functions are testing 'pinTime' but the testTweets are defining 'pinnedTime'

**Review by BRHoward — APPROVED** [review]
other than comments looks good

**Review by susanthomson — APPROVED** [review]
:norway:

**Pull request merged**